packages/react-native/Libraries/AppDelegate/RCTAppDelegate.h

```diff
@@ -58,9 +59,9 @@ NS_ASSUME_NONNULL_BEGIN
 
 /// The window object, used to render the UViewControllers
 @property (nonatomic, strong, nonnull) UIWindow *window;
-@property (nonatomic, strong, nullable) RCTBridge *bridge;
```

Comment:
Although I like that we are removing the bridge, I think we should keep the property here, not to be breaking.
Unfortunately, I have seen a bunch of code in libraries where the do stuff like `[[[UIApplication shared] applicationDelegate]bridge];`
If we remove this prop, we will break them.

Reply:
`bridge` and `bridgeAdapter` is back :)